tests/drivers/build_all/display/app.overlay

```diff
@@ -52,7 +52,7 @@
 			clock-frequency = <2000000>;
 
 			/* one entry for every devices at spi.dtsi */
-			cs-gpios = <&test_gpio 0 0>;
+			cs-gpios = <&test_gpio 0 0 &test_gpio 0 1>;
 
 			test_spi_gc9x01x: gc9x01x@1 {
```

Comment:
why not the following instead? What did I miss? 😅

```suggestion
			test_spi_gc9x01x: gc9x01x@0 {
			...
			reg = <0>;
```

Reply:
The address 0 is sort of kind of used by the `test_mipi_dbi_ili9342c`, it's just that the mipi-dsi thing has been designed to not have a node as a child of the spi bus instead. It's just a build test so this could actually use 0 but formally there's another device there.